cmd/gen-go-sample/bytes.go

```diff
@@ -14,8 +14,80 @@
 
 package main
 
+import (
+	"bytes"
+	"text/scanner"
+
+	"github.com/googleapis/gapic-generator-go/internal/errors"
+	"github.com/googleapis/gapic-generator-go/internal/pbinfo"
+)
+
 func bytesFmt() func(*generator, string) (string, error) {
 	return func(_ *generator, s string) (string, error) {
 		return "[]byte(" + s + ")", nil
 	}
 }
+
+// The information we need to generate reading all bytes of a file
+// and assign them to a local variable
+type fileInfo struct {
```

Comment:
Here and below, please use the Go commenting style: complete sentences beginning with the name of the object being described.
https://blog.golang.org/godoc-documenting-go-code